packages/web-components/src/index.ts

```diff
@@ -2,5 +2,7 @@ export * from './badge/index.js';
 export * from './counter-badge/index.js';
 export * from './progress-bar/index.js';
 export * from './text/index.js';
+export * from './accordion/accordion/index.js';
+export * from './accordion/accordion-item/index.js';
```

Comment:
Let's export these alphabetically - it's nitty but I think it'll be too difficult to keep track if things aren't in some kind of order that allows us to grok quickly.